liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/CreateServerMojo.java

```diff
@@ -37,6 +41,40 @@
      */
     private String template;
     
+    /**
+     * Packages to install. One of "all", "dependencies" or "project".
+     * 
+     * @parameter property="installAppPackages" default-value="dependencies"
+     * @readonly
+     */
+    private String installAppPackages = "dependencies";
+    
```

Comment:
I don't want to hold up the review for this one.  Just wanted to ask if possible to not duplicate parameters.